src/spikeinterface/core/channelslice.py

```diff
@@ -14,6 +14,8 @@ class ChannelSliceRecording(BaseRecording):
 
     """
 
+    _skip_properties = []
```

Comment:
SAme comments here.

Reply:
I think it's ok! For these recordings the noise levels are exactly the same

Reply:
What about removing the noise ones in the init specifically?

Reply:
It is a class attributes you cannot do this in the init.
My point is that it is better to recompute (even it is useless) the noise level in the very very very ucommon case where you 1) compute noise 2) concatenate channels (or slice) 3) compute the noise again.
Because it avoid deep deep bugs at long term, not bug in the code but bug in the processing workflow,  which very very hard to track because of this kind of small details

Reply:
done